src/components/ActivityFeed/ActivityFeedFeaturesSection.tsx

```
@@ -69,13 +69,13 @@ function FeaturesSection() {
 	};
 
 	const handleFeatureClick = (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>, feature: IFeature) => {
-		if (feature.title === 'Identity') {
+		if (feature?.title === 'Identity') {
 			e.stopPropagation();
 			e.preventDefault();
-			if (typeof currentUser?.id === 'number' && !Number.isNaN(currentUser.id) && currentUser?.username) {
+			if (typeof currentUser?.id === 'number' && !isNaN(currentUser.id) && currentUser?.username) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)